twitter-salesforce/pom.xml

```diff
@@ -106,20 +109,85 @@
         <!-- we do not want version in the JAR name -->
         <finalName>${project.artifactId}</finalName>
 
-        <plugins>
-            <plugin>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-maven-plugin</artifactId>
-                <version>${spring.boot-version}</version>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>repackage</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+				<version>${spring.boot-version}</version>
+				<configuration>
+					<systemPropertyVariables>
+						<camel.component.salesforce.clientId>${camelSalesforce.clientId}</camel.component.salesforce.clientId>
+						<camel.component.salesforce.clientSecret>${camelSalesforce.clientSecret}</camel.component.salesforce.clientSecret>
+						<camel.component.salesforce.userName>${camelSalesforce.userName}</camel.component.salesforce.userName>
+						<camel.component.salesforce.password>${camelSalesforce.password}</camel.component.salesforce.password>
+						<camel.component.salesforce.loginUrl>https://login.salesforce.com</camel.component.salesforce.loginUrl>
```

Comment:
Add https://login.salesforce.com to `<properties/>`, replace with ${salesforce.loginUrl}